### PR TITLE
chore: migrate routes folder to tsx

### PR DESCRIPTION
**Pull request by aizad-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/reports/src/Components/Routes/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import BinaryLink from './binary-link.jsx';
```
Comment (akmal-deriv) [inline packages/reports/src/Components/Routes/index.ts]: Please, update the extension here

File: packages/reports/src/Components/Routes/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+import RouteWithSubRoutes from './route-with-sub-routes.jsx';
```
Comment (akmal-deriv) [inline packages/reports/src/Components/Routes/index.ts]: Same, update the extension

File: packages/reports/src/Components/Routes/index.ts (hunk 1 above)
Comment (aizad-deriv) [reply packages/reports/src/Components/Routes/index.ts]: `BinaryLink` has not yet been migrated to tsx.

File: packages/reports/src/Components/Routes/index.ts (hunk 2 above)
Comment (aizad-deriv) [reply packages/reports/src/Components/Routes/index.ts]: `RouteWithSubRoutes ` has not yet been migrated to tsx.

**Review by suisin-deriv — APPROVED** [review]
LGTM

**Pull request closed without merging**